pages/web_cloud/domains/transfer_incoming_godaddy/guide.fr-fr.md

```diff
@@ -0,0 +1,100 @@
+>
+> Nous mettons à votre disposition ce guide afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [prestataire spécialisé](https://partner.ovhcloud.com/fr/directory/) et/ou de contacter votre bureau d'enregistrement actuel si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [Aller plus loin](#go-further) de ce guide.
+>
+
+## En pratique
+
+> [!primary]
+>
+> La zone DNS active d'un nom de domaine contient la configuration DNS appliquée à votre nom de domaine. C'est elle qui lie votre nom de domaine à vos services tels que vos adresses e-mails ou votre site web.
+>
+> Si, en complément de votre nom de domaine, vous disposez aussi d'une zone DNS active pour celui-ci chez votre bureau d'enregistrement actuel, vérifiez auprès de ses services que la zone DNS appliquée à votre nom de domaine ne va pas être supprimée une fois le transfert effectué.
+>
+> En effet, certains bureaux d'enregistrements suppriment la zone DNS présente chez eux dès que le transfert de votre nom de domaine est terminé. Si tel est le cas, recréez à l'identique votre zone DNS chez OVHcloud avant de démarrer les actions liées au transfert de votre nom de domaine.
+>
+> Pour cela, consultez les guides suivants :
+> - [Créer une zone DNS chez OVHcloud](/pages/web_cloud/domains/dns_zone_create)
+> - [Éditer une zone DNS OVHcloud](/pages/web_cloud/domains/dns_zone_edit)
+>
+> Vérifiez également que votre bureau d'enregistrement actuel ne va pas fermer d'autres services comme, par exemple, les adresses e-mails associées à votre nom de domaine.
+>
+
+### Etape 1 - Déverrouiller le nom de domaine chez GoDaddy
+
+Le verrouillage d'un nom de domaine le protège contre les tentatives de transfert non autorisées.
+GoDaddy active cette protection par défaut. Vous devez donc la désactiver pour transférer votre domaine chez OVHcloud.
+
+Pour cela : 
+
+1.	Connectez-vous à votre compte GoDaddy, rendez-vous dans votre portfolio de `Noms de domaines`{.action} et cochez la case à côté du nom de domaine à transférer.
+2.	Choisissez le menu `Verrouillage`{.action} et sélectionnez `Désactiver le verrouillage`{.action} pour déverrouiller votre nom de domaine.
+3.	Examinez le nom de domaine et cliquez sur `Continuer`{.action}.
+4.	Si vous activez la protection du domaine, vérifiez votre identité.
+5.	La notification de Succès apparaîtra pour indiquer que votre domaine est déverrouillé.
+
+### Etape 2 - Obtenir le code d’autorisation 
+
+OVHcloud vous demandera de fournir le code d’autorisation ou « Auth code » avant d’initier la procédure de transfert de nom de domaine. Vous pouvez l’obtenir en ouvrant la page de votre portfolio de `Noms de domaines`{.action} chez GoDaddy.
+
+Pour cela : 
+
+1.	Sélectionnez le nom de domaine concerné pour accéder au menu `Paramètres`{.action} du nom de domaine.
```

Comment:
Let's link the GoDaddy guide instead of the detailed steps. This will reduce the translations burden and potential errors.